sycl/test-e2e/Basic/fpga_tests/fpga_aocx_win.cpp

```diff
@@ -6,8 +6,7 @@
 //
 //===----------------------------------------------------------------------===//
 
-// REQUIRES: opencl-aot, accelerator
-// REQUIRES: system-windows
+// REQUIRES: opencl-aot, accelerator, windows
```

Comment:
`system-windows` is default `llvm-lit` feature. https://llvm.org/docs/CommandGuide/lit.html#default-features

What is the reason to add alternative spelling?

Reply:
I agree that there are features which we "inherited" from standard LLVM/clang/LIT helpers and we should continue to use them if they work for us, there is no need to re-invent them, we just need to record them as known.

Reply:
> What is the reason to add alternative spelling?

The largest part of sycl tests use "windows" instead of "system-windows", so I thought it's some legacy. I'll discard these changes.

Reply:
[d76ffef](https://github.com/intel/llvm/pull/16019/commits/d76ffef2998f93f8ced827946a76a21a2f6078fd)

Reply:
I suggest we use default llvm-lit `system-windows` for sycl tests as well. If you agree, please, open a tracker to create a follow up patch.